ams/server/src/main/java/com/netease/arctic/server/optimizing/maintainer/MixedTableMaintainer.java

```diff
@@ -318,6 +352,87 @@ private void tryClearChangeFiles(List<DataFile> changeFiles) {
         LOG.error(unkeyedTable.name() + " failed to delete change files, ignore", t);
       }
     }
+
+    private List<IcebergTableMaintainer.ExpireFiles> keyedExpiredFileScan(
```

Comment:
The keyedExpiredFileScan in the change is not accurately scanning both the change and the base. We need to use the scan provided by Mixed to perform the scan, rather than separately scanning the change and the base and simply adding them together.